### ISSUE TITLE
Build fails for timescaldb extension community version

### ISSUE THREAD

**Issue by Ajaiswal2610** [zalando/spilo#1040]
i am trying to build spilo with timescaledb community edition by setting build arg TIMESCALEDB_APACHE_ONLY=false , but build fails
throwing 

`614.8 creating symlink /usr/share/postgresql/15/contrib/oldgrants_pgq.sql -> ../../16/contrib/oldgrants_pgq.sql
614.8 creating symlink /usr/share/postgresql/15/contrib/oldgrants_pgq_node.sql -> ../../16/contrib/oldgrants_pgq_node.sql
614.8 creating symlink /usr/share/postgresql/15/contrib/pgq.sql -> ../../16/contrib/pgq.sql
614.8 creating symlink /usr/share/postgresql/15/contrib/pgq.upgrade.sql -> ../../16/contrib/pgq.upgrade.sql
614.8 creating symlink /usr/share/postgresql/15/contrib/pgq_lowlevel.sql -> ../../16/contrib/pgq_lowlevel.sql
614.8 creating symlink /usr/share/postgresql/15/contrib/pgq_node.sql -> ../../16/contrib/pgq_node.sql
614.8 creating symlink /usr/share/postgresql/15/contrib/pgq_node.upgrade.sql -> ../../16/contrib/pgq_node.upgrade.sql
614.8 creating symlink /usr/share/postgresql/15/contrib/pgq_pl_only.sql -> ../../16/contrib/pgq_pl_only.sql
614.8 creating symlink /usr/share/postgresql/15/contrib/pgq_pl_only.upgrade.sql -> ../../16/contrib/pgq_pl_only.upgrade.sql
614.8 creating symlink /usr/share/postgresql/15/contrib/pgq_triggers.sql -> ../../16/contrib/pgq_triggers.sql
614.8 creating symlink /usr/share/postgresql/15/contrib/sepgsql.sql -> ../../16/contrib/sepgsql.sql
614.9 creating symlink /usr/share/postgresql/15/contrib/uninstall_pgq.sql -> ../../16/contrib/uninstall_pgq.sql
614.9 base.sh: line 265: cd: /usr/share/postgresql/16/contrib/postgis-3.4: No such file or directory
------
Dockerfile:68
--------------------
  66 |     
  67 |     WORKDIR /builddeps
  68 | >>> RUN bash base.sh
  69 |     
  70 |     # Install wal-g
--------------------
ERROR: failed to solve: process "/bin/sh -c bash base.sh" did not complete successfully: exit code: 1`

**Comment by Ajaiswal2610** [comment]
i figured out how to fix this , you need to pass build arg PGOLDVERSION as empty string ""

**Comment by annielzy** [comment]
I am encountering the same error, and `TIMESCALEDB_APACHE_ONLY` is set to true so this argument is not related to the issue. It is likely that something external to the script has changed.

**Comment by ktalo** [comment]
Yes, the base.sh seems to install `postgresql-<PGVERSION>-postgis-3`  using apt and currently that installs 3.5 instead of 3.4.  Got things working just like @annielzy by bumping the POSTGIS_VERSION variable.
